exec/continuous.stan

```diff
@@ -234,9 +234,15 @@ parameters {
 transformed parameters {
   real dispersion;
   #include "tparameters_glm.stan" // defines beta, b, theta_L
-  if (prior_scale_for_dispersion > 0)
-    dispersion =  prior_scale_for_dispersion * dispersion_unscaled;
-  else dispersion = dispersion_unscaled;
+  
+  if (prior_dist_for_dispersion == 0 || prior_scale_for_dispersion <= 0)
```

Comment:
Is it possible for `prior_scale_for_dispersion` to be `<= 0` when `prior_dist_for_dispersion > 0`?